packages/cli/register/src/ir-to-fdr-converter/convertPackage.ts

```diff
@@ -70,25 +70,27 @@ function convertService(
     for (const irEndpoint of irService.endpoints) {
         const autogeneratedExample = irEndpoint.autogeneratedExamples[0];
         const examples: FdrCjsSdk.api.v1.register.ExampleEndpointCall[] = [];
-        if (irEndpoint.userSpecifiedExamples.length > 0 && autogeneratedExample != null) {
+        if (irEndpoint.userSpecifiedExamples.length > 0) {
             examples.push(
-                ...irEndpoint.userSpecifiedExamples.map((userSpecifiedExample) =>
-                    convertHttpEndpointExample({
-                        autogeneratedExample,
-                        userSpecifiedExample,
-                        irEndpoint,
-                        ir
-                    })
-                )
+                ...irEndpoint.userSpecifiedExamples
+                    .map((userSpecifiedExample) =>
+                        convertHttpEndpointExample({
+                            userSpecifiedExample,
```

Comment:
can you update this function to take an autogeneratedExample if present -- basically sometimes users can create partial examples and they need to be combined with the autogenerated one

Reply:
Done